src/docs/cookbook/maintenance/error-reporting.md

```diff
@@ -49,9 +49,9 @@ To get a DSN, please:
 
 ## 2. Import the Sentry package
 
-Next, we'll need to import the
-[`sentry`]({{site.pub}}/packages/sentry) package into our app. The
-sentry package will make it easier for us to send error reports to the Sentry
+Import the
+[`sentry`]({{site.pub}}/packages/sentry) package into the app. The
```

Comment:
It looks like some of the `{{site.pub}}/packages` URLs still exist (should be `{{site.pub-pkg}}`).

Should do a global search-replace... maybe in a future PR.